ansible/roles/atmo-ephemeral-mount/tasks/main.yml

```diff
@@ -20,7 +20,7 @@
   when:
     - 'ephem_stat.stat.islnk is defined'
     - 'ephem_stat.stat.islnk == true'
-    - 'item.device == "{{ ephem_stat.stat.lnk_source }}"'
+    - 'item.device == "ephem_stat.stat.lnk_source"'
```

Comment:
In order for this to work you'll need to remove the quotes around `ephem_stat...`. I believe you're comparing the device to the literal string "ephem_stat...".

You can test this in isolation of the rest of the playbook by creating an ephem_stat variable that holds a sample dictionary.